frontend/src/components/Homepage/Homepage/Homepage.tsx

```diff
@@ -26,6 +27,12 @@ const everyoneItems: readonly NavigationCardItem[] = [
     header: 'Team Event Credits',
     description: 'Track your team event credits.',
     link: '/forms/teamEventCredits'
+  },
+  {
+    header: 'Coffee Chats',
+    description: 'Submit your coffee chats.',
+    link: '/forms/coffeeChats',
+    adminOnly: !ENABLE_COFFEE_CHAT
```

Comment:
Oddly enough, I realized that this wasn't being hidden on the homepage even though you used `adminOnly` properly here, so I did some digging and have a fix for this: https://github.com/cornell-dti/idol/pull/651